repository/c.json

```diff
@@ -2687,6 +2687,17 @@
 					"tags": true
 				}
 			]
+		},{
+			"name": "CompletePHP",
+			"details": "https://github.com/desean1625/CompletePHP",
+			"author": "Desean1625",
+			"labels": ["completions","PHP","Documentation","autocomplete"],
```

Comment:
This should be changed to `["completions", "php", "documentation", "auto-complete"]` so that the package shows up in existing labels.